### PR TITLE
Update yes :)

### PR DESCRIPTION
**Pull request by Nullacron**
Added, removed, edited, etc.

### CODE REVIEW

**Comment by ShadowCodeX-debug** [comment]
Did things yes. :)

File: README.md [hunk 1]
```diff
@@ -1,339 +1,282 @@
...
-### Disable PostFX
-[Link To Original Post](https://discord.com/channels/1099468797410283540/1150655466830041128)
-```json
-{
-    "FFlagDisablePostFx": true
-}
-```
+# Rendering and Graphics
 
-### Change Rendering Mode
```
[the remark this answers is not in the document]
Comment (GoingCrazyDude) [reply README.md:15]: Keep the rendering mode section the same no need to change it

File: README.md [hunk 2]
```diff
@@ -1,330 +1,307 @@
-# ROBLOX IS KILLING LOCAL CLIENTSETTINGS LOADING
-# NO MORE CLOSET CHEATERS
-![drake](https://github.com/pizzaboxer/bloxstrap/assets/67791946/303dfde5-1968-435e-8660-249e8b6001c6)
+> [!NOTE]
+> Local Fast Flags are being patched out. Expect your flags to not work soon enough.
```
Comment (GoingCrazyDude) [inline README.md:2]: This should be "Local ClientSettings loading is being removed. Expect FastFlags to not be modifiable by the client soon."

File: README.md [hunk 3]
```diff
@@ -1,330 +1,307 @@
...
-### Disable PostFX
-[Link To Original Post](https://discord.com/channels/1099468797410283540/1150655466830041128)
-```json
-{
-    "FFlagDisablePostFx": true
-}
-```
+# Change Rendering Mode
 
-### Change Rendering Mode
```
Comment (GoingCrazyDude) [inline README.md:15]: Keep the "Change Rendering Mode" section same as the current one in the official repo.

File: README.md (hunk 2 above)
Comment (ShadowCodeX-debug) [reply README.md:2]: The word FastFlags are more known than ClientSettings. I will change it a little bit though.

File: README.md (hunk 3 above)
Comment (ShadowCodeX-debug) [reply README.md:15]: I already did this.

**Comment by ShadowCodeX-debug** [comment]
Did more things yes? :)

**Comment by ShadowCodeX-debug** [comment]
Hey @GoingCrazyDude which one looks better?

```json
{ "NewLookConcept": true }
```

Or just keep the original?

```json
{
  "OldLook": true
}

**Comment by ShadowCodeX-debug** [comment]
(It's for ones that are lone flags.

**Comment by GoingCrazyDude** [comment]
I dont like the changes. Going to update it myself.

**Comment by Nullacron** [comment]
@GoingCrazyDude Y’know if you don’t like my edits, don’t use them. (If this was code I would call you a skid.)

**Comment by GoingCrazyDude** [comment]
> @GoingCrazyDude Y’know if you don’t like my edits, don’t use them. (If this was code I would call you a skid.)

I haven't copied any of your edits idk what you're talking about
You can't call me a skid when I'm compiling publicly available information

**Comment by Nullacron** [comment]
You definitely copied the uncap fps.

**Pull request closed without merging**